### PR TITLE
Check appSettings for undefined

### LINKED ISSUE #1057: Unable to load Playback Settings page
**Describe the bug**
Screenshot from 2020-04-11 00-11-21

**Logs**
Screenshot from 2020-04-11 00-11-37

**System:**
 - OS: Arch Linux
 - Browser: Firefox, Chrome
 - Jellyfin Version: nightly

**Additional context**

This was recently built manually from the master and I cant remember the specific date.
The browser cache has been cleared.

### PR DESCRIPTION
**Pull request by MrTimscampi**
**Changes**

Prevents the infinite loading on the Playback settings by strictly checking for undefined or null instead of loosely checking for null.

**Issues**

Fixes #1057

### CODE REVIEW

File: src/scripts/settings/appSettings.js [hunk 1]
```diff
@@ -12,15 +12,15 @@ import events from 'events';
     }
 
     export function enableAutoLogin(val) {
-        if (val != null) {
+        if (val !== undefined || null) {
```
Comment (dmitrylyzo) [inline src/scripts/settings/appSettings.js:15]: `val !== undefined || null` ~ `val !== undefined || false` ~ `val !== undefined`

maybe `val !== undefined && val !== null`?

Comment (MrTimscampi) [reply src/scripts/settings/appSettings.js:15]: Hm yeah, good point. I was trying to make it shorter but that's not ideal :p

**Comment by YouKnowBlom** [comment]
I was a bit quick on the trigger. It seems `val !== null` is always true (indicated by sonarcloud). With that said, it fixed the issue :p

**Comment by dmitrylyzo** [comment]
Originally, `val != null` worked for
- `enableAutoLogin()`
- `enableAutoLogin(undefined)`
- `enableAutoLogin(null)`

I believe we never do this, but what if someone calls `enableAutoLogin(null)`?

**Comment by dkanada** [comment]
I think that would be overengineering in this case.

**Pull request merged**